wowup-electron/src/app/pages/home/home.component.ts

```diff
@@ -74,19 +64,17 @@ export class HomeComponent implements AfterViewInit, OnDestroy {
 
     this.initAppUpdateCheck();
 
-    this._warcraftService.installedClientTypes$
-      .pipe(
-        first((clientTypes) => !!clientTypes),
-        switchMap((clientTypes) => from(this.migrateAddons(clientTypes)))
-      )
+    this._warcraftInstallationService.wowInstallations$
+      .pipe(first((installations) => installations.length > 0))
+      // TODO process migrations
```

Comment:
Should this TODO still be done?

Reply:
Hmm i think thats old ,  currently legacy installs a migrated in the service layer now not the ui layer.

addonService:
this._warcraftInstallationService.legacyInstallationSrc$